c3s_eqc_automatic_quality_control/dashboard.py

```diff
@@ -63,10 +67,14 @@ def set_logfile(logger: logging.Logger, logfilepath: pathlib.Path) -> logging.Lo
 
 
 def ensure_log_dir() -> pathlib.Path:
-    log_dir = pathlib.Path.home() / ".eqc/logs/"
-    if not log_dir.is_dir():
-        log_dir.mkdir(parents=True, exist_ok=True)
-    return log_dir
+    if os.environ.get(EQC_AQC_ENV_VARNAME) is None:
+        eqc_dir = pathlib.Path.home() / ".eqc"
+    else:
+        eqc_dir = pathlib.Path(os.environ[EQC_AQC_ENV_VARNAME])
+    log_dir_path = eqc_dir / "logs/"
+    if not log_dir_path.is_dir():
+        log_dir_path.mkdir(parents=True, exist_ok=True)
+    return log_dir_path
```

Comment:
I think you can get rid of the if statements as pathlib takes care of it:
```suggestion
    eqc_dir = os.environ.get(EQC_AQC_ENV_VARNAME, pathlib.Path.home() / ".eqc")
    log_dir_path = pathlib.Path(eqc_dir) / "logs"
    log_dir_path.mkdir(parents=True, exist_ok=True)
    return log_dir_path
```